addons/website/static/src/scss/website.scss

```diff
@@ -1122,6 +1122,33 @@ table.table_desc tr td {
     }
 }
 
+// Navbar templates
+.o_navbar_showcase{
```

Comment:
Quick comment before I check deeper: add a space before `{` and basically follow this: https://github.com/odoo/odoo/wiki/SCSS-coding-guidelines (even though I did not double check those guidelines for a while...